File: python/count_repetition/count_repetition.py
def count_repetition(source):
  count = 0
  map = {}

  for number in source:
    value = map.get(number)
    
    if value is not None:
      map[number] = value + 1
    else:
      map[number] = 0

  for key, value in map.items():
    count += value

  return count

File: php/count_repetition/src/CountRepetition.php
<?php

namespace App;

class CountRepetition
{
  public static function countRepetition(array $source): int
  {
    $map = array();

    foreach ($source as $number) {
      if (isset($map[$number])) {
        $map[$number] = $map[$number] + 1;
      } else {
        $map[$number] = 0;
      }
    }

    return array_sum($map);
  }
}
